scripts/install.sh

```diff
@@ -281,6 +282,7 @@ kubectl get secret storage-config
 info "Installing ModelMesh Serving RBACs (namespace_scope_mode=$namespace_scope_mode)"
 if [[ $namespace_scope_mode == "true" ]]; then
   kustomize build rbac/namespace-scope | kubectl apply -f -
+  sed -i 's/- bases\/serving.kserve.io_clusterservingruntimes.yaml/#- bases\/serving.kserve.io_clusterservingruntimes.yaml/g' crd/kustomization.yaml
```

Comment:
Ah I see, you didn't make a separate kustomization for this. I guess this is ok. Maybe add a comment above this line saying that we won't install the CSR CRD in namespace mode?